vllm/core/scheduler.py

```diff
@@ -427,7 +427,6 @@ def _schedule_running(
                         swapped_out.append(seq_group)
                     break
             else:
-                logger.debug(f"append slot for {seq_group}")
```

Comment:
this was the major issue

Reply:
This adds 5~10us overhead to this logic, which ends up making 2ms scheduler iteration overhead to 10+ms 😮